### PR TITLE
R20.2 407

### PR DESCRIPTION
**Pull request by dougkenik**
Implemented functionality to toggle on/off results buttons and associated data.
Fixed a bug in failed optimizations for viewing displacement results

### CODE REVIEW

File: SmartSlicePlugin/SmartSliceCloudProxy.py [hunk 1]
```diff
@@ -843,6 +843,7 @@ def showProblemMeshes(self):
 
     @pyqtSlot()
     def hideProblemMeshes(self):
+        our_only_node =  getPrintableNodes()[0]
```
Comment (bradymadams) [inline SmartSlicePlugin/SmartSliceCloudProxy.py:846]: Is this necessary?

**Review by bradymadams — COMMENTED** [review]
Does this close any open issues?

File: SmartSlicePlugin/SmartSliceCloudProxy.py (hunk 1 above)
Comment (dougkenik) [reply SmartSlicePlugin/SmartSliceCloudProxy.py]: Nope

**Comment by dougkenik** [comment]
Closes #407 and also fixes a bug that has not been reported yet.

**Review by bradymadams — APPROVED** [review]

**Review by jake-kirkland-teton — APPROVED** [review]

**Pull request merged**